packages/calcite-components/src/components/alert/alert.scss

```diff
@@ -140,8 +144,8 @@ $border-style: 1px solid var(--calcite-color-border-3);
 
 .text-container {
   @apply box-border flex flex-auto min-w-0 flex-col break-words;
-  padding-block: var(--calcite-alert-spacing-token-small);
-  padding-inline: var(--calcite-alert-spacing-token-large) var(--calcite-alert-spacing-token-small);
+  padding-block: var(--calcite-internal-alert-spacing-token-small);
```

Comment:
Rename these to not use small medium

Reply:
There's currently `small` and `large`. They are both assigned different theme spacing based on the selected `scale`. 

`--calcite-internal-alert-spacing-token-small: theme("spacing.2");
  --calcite-internal-alert-spacing-token-large: theme("spacing.3");`

What names should be applied instead of small and large?